cli/filelister.cpp

```diff
@@ -95,6 +99,17 @@ void FileLister::addFiles(std::map<std::string, std::size_t> &files, const std::
             searchPattern += '*';
             basedir = cleanedPath;
             break;
+        case 'c':   // ends with *.c
+        case 'p': { // ends with *.cpp
+            const std::string::size_type pos = searchPattern.find_last_of('\\');
```

Comment:
How can there be a `\\` in the search pattern? The `Path::fromNativeSeparators` should convert given paths because internally I am expecting that the path separator is always '/'.